src/main/java/com/microsoft/jenkins/keyvault/SecretCertificateCredentials.java

```diff
@@ -41,10 +41,10 @@
     public SecretCertificateCredentials(CredentialsScope scope,
                                         String id,
                                         String description,
-                                        String credentialId,
+                                        String servicePrincipalId,
```

Comment:
Without this change creating a secret string credential via the UI didn't seem to work at all =/, unrelated to the PR but picked it up while doing interactive testing.